src/containers/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { content } from '../../data/LanguageContent';
+
+import LanguageDropdown from '../../components/LanguageDropdown/LanguageDropdown';
+import Video from '../../components/Video/Video';
+import HubspotForm from '../../components/HubspotForm/HubspotForm';
+
+import './MainContainer.css';
+
+const MainContainer = () => {
+  const [language, setLanguage] = useState('en');
+  const browserLanguage = window.navigator.userLanguage || window.navigator.language;
+   
+  useEffect(() => {
+    if (browserLanguage){
```

Comment:
do we need the if (browserLanguage) ? since alrready in the userEffect and has the dependency array at the end then we should be ok right?